kolena/_experimental/instance_segmentation/dataset.py

```diff
@@ -11,26 +11,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Dict
+from typing import Literal
+from typing import Union
+
+import pandas as pd
+
 from kolena._experimental.object_detection.dataset import upload_object_detection_results
 
 
-upload_instance_segmentation_results = upload_object_detection_results
```

Comment:
Unfortunately this was the only way I could get mkdocstring to pick up the alias